go/pkg/bertymessenger/db.go

```diff
@@ -124,6 +124,18 @@ func getInteraction(db *gorm.DB, cid string) (Interaction, error) {
 	return interactions[0], nil
 }
 
+func getMember(db *gorm.DB, publicKey string) (Member, error) {
+	var members []Member
+	err := db.Where(&Member{PublicKey: publicKey}).Find(&members).Error
+	if err != nil {
+		return Member{}, err
+	}
+	if len(members) == 0 {
```

Comment:
very good way to avoid having an error in the logs :+1: 

however, can you add a check that "should" never happen: `if len(members) > 1` so you can raise an error instead of ignoring this strange thing?